plugin.video.rocketbeans/resources/settings.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8" standalone="yes"?>
+<settings>
+    <setting id="stream" type="select" label="Livestream Quelle" values="YouTube|Twitch" default="YouTube" />
```

Comment:
Can you please add this to a language file to avoid hardcoded strings? Or at least use English? :)

Reply:
Sure, probably even qualifies as an oversight. Even as there is only german content, I've keept everything english until now.

Reply:
New version pushed